database/factories/CharacterFactory.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\User;
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Illuminate\Support\Str;
+use App\Models\Character;
+
+class CharacterFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = Character::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            'user_id' => User::factory(),
+            'name' => $this->faker->name(),
+            'race' => $this->faker->word(),
+            'class' => $this->faker->word(),
```

Comment:
Is there a way to create values for these fields from an `enum`?

Reply:
Yes, there is, but iirc we resolved that since there is the possibility that users can have special races, we left this as strings. 

Classes were left as strings because characters can do something called "multi-classing" which means they may have an an arbitrary amount of classes on a character